manifests/charts/istio-cni/templates/network-attachment-definition.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if eq .Values.provider "multus" }}
+{{- if eq .Values.global.platform "openshift" }}
```

Comment:
Can you not use multus off openshift..?

Reply:
The only place where `multus` is currently used is where we set things to work on openshift. @dgn I think you were the last one to make some changes here. If we are using `provider=multus` and `platform=openshift` to set params that are only used on openshift deploy make sense to merge in only one.

Reply:
Umm, ok, I just saw this [PR](https://github.com/istio/istio/pull/49495) from Daniel. Not so long ago, the provider was added to replace chained. @dgn do you have any opinion into change this or we should leave it as `provider`

Reply:
Multus is a generic CNI that can be used on any k8s platform, no? https://github.com/k8snetworkplumbingwg/multus-cni

Reply:
Yes, apologies if I was not clear. Multus, yes, is something generic, but we are using provider for now only to set params that are related to openshift profiles. But, as you mentioned is something generic, so, I changed my mind and think also that is better to leave it as it is and not replace the use of `pilot.cni.provider` since can be used in the future for others purposes

Reply:
You can use it off of openshift, yes. I don't know of anyone who does, and this was added for (and is only tested against) openshift  anyway.

How we code around the use of `multus` in our charts is entirely `openshift` specific, so if you just set `provider=multus` and did not use or also set up openshift, the resulting config would almost certainly not be valid.

We don't test, document or validate the use of `multus` outside of openshift, so it's moot.

So, I'd prefer to drop it, if the openshift people are amenable.

Reply:
> since can be used in the future for others purposes

If the consensus is we should keep an unused, untested, and unsupported Helm value around for hypothetical future use cases, that is fine - but we should still remove it from all the chart conditionals where it is currently redundant with `platform`, to simplify those, and to indicate we do not test or support multus outside of openshift ATM.

Should that change, someone can actually do that work.